languages/js/src/Host.ts

```diff
@@ -158,6 +205,26 @@ export class Host {
     return instanceId;
   }
 
+  /**
+   * Register the MROs of all classes that are registered on `this`.
+   */
+  registerMros() {
+    // Get MRO of all registered classes
+    // NOTE: not ideal that the MRO gets updated each time loadStr is
+    // called, but since we are planning to move to only calling load once
+    // with the include feature, I think it's okay for now.
+    for (const typ of this.distinctUserTypes()) {
+      // Get MRO for type.
+      const mro = ancestors(typ.cls)
+        // TODO: as conversion ok?
```

Comment:
Is this TODO still relevant?

Reply:
Yeah... this coerces `c` which is a Function into the `Class` type. This is because `ancestors` returns an iterable of `Function`s, not the `Class` type that we've created.

Reply:
Not sure this is ok, could use @gj 's input

Reply:
we talked — it's cool for now